### PR TITLE
🙄

### PR DESCRIPTION
**Pull request by mlondschien**

### CODE REVIEW

File: _posts/2024-03-11-getting-codesigning-to-work-on-apple-silicon.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ title: >
...
+If your error message is simply "Killed: 9", you know you are in for a long debug session.
 Sadly, in this case, even a debugger would not give you any further information.
-Thus, this has been a problem that could only be fixed by having good insight knowledge the packaging percularities of the particular platform.
+Thus, this has been a problem that could only be fixed by having good insight and knowledge of the packaging percularities of the particular platform.
 
-One of these programs that failed with the above mentioned error message was `conda-pack`.
+One of these programs that failed with the above-mentioned error message was `conda-pack`.
 `conda-pack` is a tool to package `conda` environments into an archive that can be extracted later on at an arbitrary location.
-Through a specialised activation script, the executable in the packed environment can then from this arbitrary location.
+Through a specialised activation script, the executable in the packed environment can then ??? from this arbitrary location.
```
Comment (mlondschien) [inline _posts/2024-03-11-getting-codesigning-to-work-on-apple-silicon.md]: ???

**Comment by pavelzw** [comment]
The date is also incorrect 😔 

typos pre-commit when?

File: _posts/2024-03-11-getting-codesigning-to-work-on-apple-silicon.md (hunk 1 above)
Comment (xhochy) [reply _posts/2024-03-11-getting-codesigning-to-work-on-apple-silicon.md]:
```suggestion
Through a specialised activation script, the executable in the packed environment can then be run from this arbitrary location.
```

**Comment by xhochy** [comment]
🥰

**Pull request merged**